### PR TITLE
pascal casing component filenames

### PR DESCRIPTION
**Pull request by teallarson**
## Change description

All front end component filenames are now PascalCase to follow React's name casing convention.

### Development

- [ ] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

**Review by krishnaglick — APPROVED** [review]
I do not envy the work done here, but I'm glad we're moving to more standardized conventions!

File: components/home/IntegrationInstallation.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { Container, Row, Col } from "react-bootstrap";
 import Link from "next/link";
-import HomepageCodeBox from "../../components/home/homepageCodeBox";
+import HomepageCodeBox from "./HomepageCodeBox";
```
Comment (evantahler) [inline components/home/IntegrationInstallation.tsx:3]: nice - some simpler imports

File: components/solutions/ComparisonPage.tsx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 import Head from "next/head";
-import ComparisonTable from "./comparisonTable";
-import FeatureBanner from "./featureBanner";
+import ComparisonTable from "./ComparisonTable";
+import FeatureBanner from "./FeatureBanner";
 import hydrate from "next-mdx-remote/hydrate";
 import { ComparisonInfo } from "../../utils/solutionsPages";
 import { arrayToReadableList, possessiveNoun } from "../../utils/inflectors";
-import * as components from ".";
+import * as components from "./Index";
```
Comment (evantahler) [inline components/solutions/ComparisonPage.tsx:9]: 👍 - this is probably a clearer import statement than the magic "index = default"

**Review by evantahler — APPROVED** [review]
💥

File: components/home/IntegrationInstallation.tsx (hunk 1 above)
Comment (teallarson) [reply components/home/IntegrationInstallation.tsx:3]: That fixed itself when I let VSCode auto-correct the import.  Thanks VSCode!

**Pull request merged**